docs/index.yml

```diff
@@ -99,7 +109,7 @@ conceptualContent:
         - url: core/sdk.md
           itemType: download
           text: Download the .NET Core SDK
-        - url: core/tools/index.md
+        - url: https://dotnet.microsoft.com/download
```

Comment:
Was this supposed to go on line 109 @wadepickett ?

Reply:
Just a note so I don't think I forgot to reply on this one. The fix and discussion for this moved to #18159.  Thanks @gewarren!